src/odemis/acq/fastem_conf.py

```diff
@@ -145,20 +145,28 @@ def configure_scanner(scanner, mode):
         raise ValueError("Invalid mode %s." % mode)
 
 
-def configure_detector(detector, roc):
+def configure_detector(detector, rocs):
     """
     Configure the detector by setting the calibrated parameters as stored for
     the provided region of calibration (ROC). If calibrated parameters are not
     available, the current settings on the detector will stay as is.
     :param detector: (technolution.MPPC) The detector to be configured.
-    :param roc: (FASTEMROC) The region of calibration as selected on the scintillator,
-                which stores the calibrated settings if calibration was performed.
+    :param rocs: (list of FastEMROC) The region of calibration as selected on the scintillator,
+                 which stores the calibrated settings if calibration was performed.
     """
-    if "cellDarkOffset" in roc.value.parameters:
-        detector.cellDarkOffset.value = roc.value.parameters["cellDarkOffset"]
-    else:
+    # check all parameters are available
+    if not any(roc.parameters and "cellDarkOffset" in roc.parameters for roc in rocs):
```

Comment:
doesn't this still log the warning when you only pass the cell translation ROC?

Reply:
Yes correct - in that case you want to have the warning that the following acquisition does not have a calibrated dark offset I would assume. What do you think?

Reply:
Yes indeed, I thought this was somehow also called outside the acquisition. But that's not true, so this behavior is correct.